packages/remote-feature-flag-controller/src/remote-feature-flag-controller.ts

```diff
@@ -291,4 +327,50 @@ export class RemoteFeatureFlagController extends BaseController<
   disable(): void {
     this.#disabled = true;
   }
+
+  /**
+   * Sets a local override for a specific feature flag.
+   *
+   * @param flagName - The name of the feature flag to override.
+   * @param value - The override value for the feature flag.
+   */
+  setFlagOverride(flagName: string, value: Json): void {
+    this.update(() => {
+      return {
+        ...this.state,
+        localOverrides: {
+          ...this.state.localOverrides,
+          [flagName]: value,
+        },
+      };
+    });
+  }
+
+  /**
+   * Clears the local override for a specific feature flag.
+   *
+   * @param flagName - The name of the feature flag to clear.
+   */
+  clearFlagOverride(flagName: string): void {
```

Comment:
Saying that we are "clearing" the override could be a bit confusing, perhaps we want to say that we are removing it?

``` suggestion
  removeFlagOverride(flagName: string): void {